tests/e2e/replication/show.py

```diff
@@ -30,14 +30,14 @@ def test_show_replicas(connection):
     cursor = connection(7687, "main").cursor()
     actual_data = set(execute_and_fetch_all(cursor, "SHOW REPLICAS;"))
 
-    expected_column_names = {"name", "socket_address", "sync_mode", "timeout"}
+    expected_column_names = {"name", "socket_address", "sync_mode", "timeout", "state"}
     actual_column_names = {x.name for x in cursor.description}
     assert expected_column_names == actual_column_names
 
     expected_data = {
-        ("replica_1", "127.0.0.1:10001", "sync", 0),
-        ("replica_2", "127.0.0.1:10002", "sync", 1.0),
-        ("replica_3", "127.0.0.1:10003", "async", None),
+        ("replica_1", "127.0.0.1:10001", "sync", 0, "ready"),
+        ("replica_2", "127.0.0.1:10002", "sync", 1.0, "ready"),
+        ("replica_3", "127.0.0.1:10003", "async", None, "ready"),
```

Comment:
Maybe try to add at least an invalid state to make sure it works. I guess a non-existing replica would do the trick, but if not, then feel free to leave as is, I can check it manually.

Reply:
I have adapted the test:
-We drop a replica and check it is not anymore in the list. 
-We kill the two other replicas and check that their state is invalid.